cpp/core/jni/JniWrapper.cc

```diff
@@ -908,18 +907,6 @@ Java_io_glutenproject_memory_alloc_NativeMemoryAllocator_bytesAllocated(JNIEnv*
   JNI_METHOD_END(-1L)
 }
 
-JNIEXPORT void JNICALL Java_io_glutenproject_tpc_MallocUtils_mallocTrim(JNIEnv* env, jobject obj) {
```

Comment:
After search all project about this interface, I can not find any usage.

Reply:
@zhztheplayer Can we remove the API?

Reply:
+1, it should be removed.